docker/Dockerfile

```diff
@@ -1,56 +1,49 @@
-FROM centos:8.4.2105
+FROM quay.io/centos/centos:stream9
 
 MAINTAINER IoT team
 
 ENV DB_HOST localhost
 ENV KEYSTONE_ADMIN_PASSWORD 4pass1w0rd
-ENV KEYSTONE_SCIM_VERSION 1.7.0
-ENV KEYSTONE_SPASSWORD_VERSION 1.18.0
+ENV KEYSTONE_SCIM_VERSION 1.8.0
+#ENV KEYSTONE_SPASSWORD_VERSION 1.19.0
```

Comment:
Should be uncommented?

Reply:
Done in https://github.com/telefonicaid/fiware-keystone-spassword/pull/238/commits/d0de6fb5b6b931ae5177b7cab93e9d29c19954da